README.md

```diff
@@ -93,13 +95,14 @@ and is available at large cloud providers.
 
 ### Supported database engine versions
 
-Operating System | MariaDB | PostgreSQL | SQLite
----------------- | --------| -----------|------------------------
-Debian 11        | 10.5    | 13.3       | 3.34
-Docker           | n/a     | n/a        | n/a
-FreeBSD 13.0     | 5.7     | 13.5       | 3.35.5
-Ubuntu 20.04     | 10.3    | 12.4       | 3.31.1
-Rocky Linux 8.4  | 10.3    | 10.17      | 3.26
+Operating System | MariaDB | PostgreSQL      | SQLite
+---------------- | --------| ----------------|------------------------
+CentOS Linux 7   | 5.5     | *not supported* | 3.7.17 ?
+Debian 11        | 10.5 ?  | 13.3 ?          | 3.34 ?
```

Comment:
For Debian,
MariaDB: 10.5, PostgreSQL 13.5, SQLite 3.34

Reply:
Fixed.